src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -20,4 +23,33 @@ public LineResponse saveLine(LineRequest request) {
         Line persistLine = lineRepository.save(request.toLine());
         return LineResponse.of(persistLine);
     }
+
+    @Transactional(readOnly = true)
+    public List<LineResponse> findAllLines() {
+        List<Line> Lines = lineRepository.findAll();
+
+        return Lines.stream()
+                .map(station -> LineResponse.of(station))
```

Comment:
```suggestion
            .map(LineResponse::of)
```

람다의 메서드 참조 형식을 사용할 수 있을 것 같아요 !

Reply:
StationService 를 참고하면서 만들면서 복불의 폐허.. 부끄럽네요.. 수정하겠습니다.